addons/ophthalmology/functions/fnc_handleRespawn.sqf

```diff
@@ -0,0 +1,22 @@
+#include "..\script_component.hpp"
+/*
+ * Author: MiszczuZPolski
+ * Ensures proper initial values reset on respawn
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Corpse <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [alive, body] call kat_misc_fnc_handleRespawn;
+ *
+ * Public: No
+ */
+
+params ["_unit","_dead"];
+TRACE_2("handleRespawn",_unit,_dead);
+
+[_unit] call FUNC(fullHealLocal);
```

Comment:
why calling fullHealLocal on respawn?

Reply:
because fullHealLocal stores all default variables, this way its easier to keep track of it